data/competencies.yml

```diff
@@ -1188,3 +1188,17 @@
   level: senior2
   area: leadership
   domain: null
+
+# S2
+- id:
+  summary: Actively encourages and shapes the personal development of more junior people
+  examples:
+    - Helps other engineers to form or meet specific career goals
+    - Organises conference attendance for a group of engineers
+    - Considerately gives targeted feedback with examples of a behaviour and its impact
```

Comment:
This example could be a little controversial. We have existing Engineer competencies around feedback, e.g. 
```
Regularly gives timely actionable feedback to colleagues
``` 
but nothing at more senior levels. I'd like to build on this because I've noticed that S2 Engineers tend to give more constructive feedback and more feedback which is targeted towards career growth or meeting engineering competencies. Is there a way of capturing this better?

Reply:
I really like how it's phrased